### ISSUE TITLE
More user-friendly navigation

### ISSUE THREAD

**Issue by raymondtsung** [ClusterM/hakchi2#617]
Whenever you select a folder or a game the system will remember it's position (blue border). This is actually a good thing if you don't have folders. But when you do and you're opening a folder. The user might feel lost because the first game is not starting at the beginning. This also results in the back button switching from position everytime. Which doesn't feel consistent and might confuse the user.

So by force the first game/folder starting at beginning everytime you select a folder or bootup the system and positioning the blue border at the beginning will improve it's user friendlyness.

Hopefully this is possible to do.

**Comment by Drakonas** [comment]
I'm not sure if this is related, but I am having an issue with folders that have very few games where when I move left to right, when I select the 2nd to last game, it refocuses the list far to the right causing weird shifting. The back button seems to shift from the left to the right of the list and back. It is all very wonky.

**Comment by raymondtsung** [comment]
Yes thats another issue from what I understand  can't be fixed. You need to have atleast 12 folders/games.

https://github.com/ClusterM/hakchi2/issues/416
